cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -1222,6 +1372,21 @@ Replaced by:
 ----
 ON HOME GRAPH
 ----
+
+a|
+label:syntax[]
+label:deprecated[]
+[source, cypher, role="noheader"]
+----
+MATCH (a) RETURN (a)--()
+----
+a|
+Pattern expressions producing lists of paths are deprecated, but they can still be used as existence predicates, for example in `WHERE` clauses.
+Instead, use a pattern comprehension:
+[source, cypher, role="noheader"]
+----
+MATCH (a) RETURN [p=(a)--() \| p]
+----
```

Comment:
Was this cherry picked to `4.4` ???

Reply:
Yes in https://github.com/neo4j/neo4j-documentation/pull/1558/files and https://github.com/neo4j/neo4j-documentation/pull/1559/files